src/openforms/dmn/contrib/camunda/plugin.py

```diff
@@ -122,3 +124,85 @@ def get_definition_xml(definition_id: str, version: str = "") -> str:
 
     def check_config(self):
         check_config(self)
+
+    def _get_inputs(self, xml, definition_id: str) -> list[InputElement]:
+        decision_table = xml.find(
```

Comment:
I would move the actual implementation of this into its own module, e.g. `parser.py` and add a bit of an abstraction:

Even better would be to implement that in our django-camunda library :)

```py
class Parser:
    def __init__(self, xml: str):
        self.xml = xml

    def extract_inputs(self) -> DMNInputParameter:
        ...

    def extract_outputs(self) -> DMNOutputParameter:
        ...
```

You can then leverage the LXML tree as a cached attribute in the parser instance while keeping the code to process/extract information well organized.

Reply:
https://github.com/maykinmedia/django-camunda/pull/12